packages/ag-grid-theme/css/uitk-ag-theme.scss

```diff
@@ -0,0 +1,229 @@
+@import "./uitk-ag-theme-mixin";
+
+.uitk-light .ag-theme-uitk-high,
+.uitk-light .ag-theme-uitk-medium,
+.uitk-light .ag-theme-uitk-low,
+.uitk-light .ag-theme-uitk-touch {
+  --uitk-ag-foreground: #2a2c2f;
```

Comment:
We start foundations and characteristics with --uitk-<characteristic name>- .. it may be better to have these as --uitkAgGrid like how other components tokens start

Reply:
Done